lti_consumer/templates/html/student.html

```diff
@@ -1,18 +1,23 @@
-<h2 class="problem-header">
-    ## Translators:  "External resource" means that this learning module is hosted on a platform external to the edX LMS
-    ${display_name} (External resource)
-</h2>
+% if display_header:
+    <h2 class="problem-header">
+        ## Translators:  "External resource" means that this learning module is hosted on a platform external to the edX LMS
+        ${display_name} (External resource)
```

Comment:
Also, either the entire template should be made safe by default, or you should temporarily add HTML escaping filters like `${_('...') | h}` on translations.  

See more details here: http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/safe_templates.html#making-legacy-mako-templates-safe-by-default